app/dashboard/models.py

```diff
@@ -5560,6 +5562,11 @@ def investigate_sybil(instance):
             total_sybil_score -= 1
             htmls.append('(REDEMPTIONx1)')
 
+        htmls.append(f'Google Verified: {instance.is_google_verified}')
```

Comment:
@apbendi Can you check if this is correct?

Reply:
https://travis-ci.org/gitcoinco/web.svg?branch=master